packages/core/src/server.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Context, Binding, Constructor} from '@loopback/context';
+
+/**
+ * Defines the requirements to implement a Server for LoopBack applications:
+ * start() : Promise<void>
+ * stop() : Promise<void>
+ * It is recommended that each Server implementation creates its own child
+ * Context, which inherits from the parent Application context. This way,
+ * any Server-specific bindings will remain local to the Server instance,
+ * and will avoid polluting global module scope.
```

Comment:
I find "module scope" misleading - the parent scope is at application-level. A module is allowed to have more than one application, there is no global module-level scope.